lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runtime/azbatch/util/command/LinuxCommandBuilder.java

```diff
@@ -65,4 +65,15 @@ protected String getDriverClasspath() {
   protected String getEvaluatorShimClasspath() {
     return StringUtils.join(super.classpathProvider.getEvaluatorClasspath(), CLASSPATH_SEPARATOR_CHAR);
   }
+
+  @Override
+  public String getIpAddressFilePath() {
+    return "$AZ_BATCH_JOB_PREP_WORKING_DIR/hostip.txt";
+  }
+
+  @Override
+  public String captureIpAddressCommandLine() {
+    String filePath = getIpAddressFilePath();
```

Comment:
final

Reply:
Fixed in iteration 22.

---
In reply to: [207102722](https://github.com/apache/reef/pull/1478#discussion_r207102722) [](ancestors = 207102722)